src/ota.c

```diff
@@ -367,6 +368,7 @@ enum golioth_status golioth_ota_download_component(struct golioth_client *client
                                  "",
                                  component->uri,
                                  GOLIOTH_CONTENT_TYPE_OCTET_STREAM,
+                                 NULL,
```

Comment:
```suggestion
                                next_block_idx,
```

Reply:
Yeah, that's a rebase error. It's correct on this commit, but should be changed on the next commit when the param is added to the parent function. Thanks for catching!